string target_type

---

int64 code
string message
temoto_2/Error[] error_stack
